lesson14/js/index.js

```diff
@@ -0,0 +1,83 @@
+const modalBtn = document.getElementById('js-modal-btn');
+const requestBtn = document.getElementById('js-request-btn');
+const modal = document.getElementById('js-modal');
+const ul = document.getElementById('js-ul');
+
+function addLoading() {
+  const wrapper = document.getElementById('js-wrapper');
+  const loading = document.createElement('img');
+  loading.src = "./img/loading-circle.gif";
+  loading.id = "loading";
+  wrapper.appendChild(loading);
+}
+
+function removeLoading() {
+  const loading = document.getElementById('loading');
+  loading.remove();
+}
+
+async function fetchErrorHandling(response) {
+  if (!response.ok) {
+    throw new Error("サーバーエラーが発生しました");
+  } else {
+    return await response.json();
+  }
+}
+
+async function fetchData() {
+  const response = await fetch("https://jsondata.okiba.me/v1/json/fR9k5211007080400");
+  const json = await fetchErrorHandling(response);
```

Comment:
IMO
fetch is done already.
Function name is incorrect. 
I think this is better. `getJsonOrError`

Reply:
Thank you very much.
I will use the function name you provided.

f44603b

Reply:
@sae-github  learned a lot. Thank you.👏